### PR TITLE
add tests

### PR DESCRIPTION
**Pull request by raftx24**
add tests
I added a test guard in Service Provider because if we enable it in the test mode all tests will be failed.

### CODE REVIEW

File: src/app/Events/Updated.php [hunk 1]
```diff
@@ -94,7 +94,7 @@ private function readRelation($relation, $value)
         $class = key($relation->toArray());
         $attribute = $relation->get($class);
 
-        return $class::find($value)->{$attribute};
+        return optional($class::find($value))->{$attribute};
```
Comment (aocneanu) [inline src/app/Events/Updated.php:97]: why do we need optional? Are you considering relations on other attributes than ids?

File: composer.json [hunk 2]
```diff
@@ -27,7 +27,8 @@
     },
     "autoload": {
         "psr-4": {
-            "LaravelEnso\\ActivityLog\\": "src/"
+            "LaravelEnso\\ActivityLog\\": "src/",
+            "LaravelEnso\\ActivityLog\\Test": "tests/"
```
Comment (aocneanu) [inline composer.json:35]: why do we need this?

File: src/app/Events/Updated.php (hunk 1 above)
Comment (raftx24) [reply src/app/Events/Updated.php:97]: if the relation changed from null to something or from something to null this line produce an error.

File: composer.json (hunk 2 above)
Comment (raftx24) [reply composer.json:35]: I added the namespace for tests because preventing from class name collision

Comment (aocneanu) [reply composer.json:35]: The class collision can only happen when testing but we can use prefixes for that, right?

I can't see any benefit for autoloading the tests namespaces when env !== 'testing'.

Comment (raftx24) [reply composer.json:35]: your are right, I made a mistake! it can be used in autoload-dev or use prefixes, I fix it and push it again

**Pull request merged**